hera_cal/lstbin.py

```diff
@@ -604,8 +597,15 @@ def lst_bin_files(data_files, dlst=None, verbose=True, ntimes_per_file=60, file_
     miriad_kwargs['outdir'] = outdir
     miriad_kwargs['overwrite'] = overwrite
  
-    # get frequency, time and antennas position information from the zeroth data file
+    # get metadata from the zeroth data file
     d, fl, ap, a, f, t, l, p = io.load_vis(data_files[0][0], return_meta=True, pick_data_ants=False)
+
+    # push time and lst arrays forward by half an integration
+    abscal.echo("pushing time and lst arrays forward by half an integration b/c pyuvdata "\
+                "currently does not do this for us....", verbose=verbose)
+    t += np.median(np.diff(t)) / 2.0
```

Comment:
These kind of medians of diffs will give funny answers (nans, etc.) when nTimes is 1. I'd probably add in a statement that `assert(len(t) > 1)`.